app/src/main/java/nodomain/freeyourgadget/gadgetbridge/impl/GBDevice.java

```diff
@@ -78,6 +78,15 @@ public GBDevice createFromParcel(Parcel source) {
     private String mBusyTask;
     private List<ItemWithDetails> mDeviceInfos;
 
+    private int mStep = 0;
```

Comment:
@jpbarraca is right -- this is device specific and should not be there. However, there is already support for adding "any kind" of additional information to GBDevice. See method GBDevice#addDeviceInfo(). Can you try to use that?